app/src/main/java/com/example/bigowlapp/model/Schedule.java

```diff
@@ -1,5 +1,149 @@
 package com.example.bigowlapp.model;
 
+import com.google.firebase.Timestamp;
+import com.google.firebase.firestore.DocumentId;
+import com.google.firebase.firestore.GeoPoint;
+import com.google.firebase.firestore.IgnoreExtraProperties;
+
+import java.util.List;
+import java.util.Map;
+
+@IgnoreExtraProperties
 public class Schedule {
-    // TODO: Create schedule model
-}
+
+    @DocumentId
+    private String uId;
+    private String groupUId;
+    private Timestamp startTime;
+    private Timestamp endTime;
+    private GeoPoint location;
+    private String groupSupervisorUId;
+    private Map<String, UserResponse> members;
```

Comment:
Naming is a little unclear and need refactoring.

The variable should be rename to userReponseMap.